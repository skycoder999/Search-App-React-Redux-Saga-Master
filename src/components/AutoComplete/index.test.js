import { Provider } from "react-redux";
import { AutoComplete } from "./index";
import configureStore from "../../utils/store";

let wrapper;
const store = configureStore();

beforeEach(() => {
  wrapper = shallow(
    <Provider store={store}>
      <AutoComplete />
    </Provider>
  );
});

describe("AutoComplete Component", () => {
  it("renders without crashing", () => {
    expect(wrapper.exists()).toBe(true);
  });

  it("matches the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
