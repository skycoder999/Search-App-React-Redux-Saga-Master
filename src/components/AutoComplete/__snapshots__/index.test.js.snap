// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoComplete Component matches the snapshot 1`] = `ShallowWrapper {}`;
